src/number_theory/padics/padic_integers.lean

```diff
@@ -140,13 +142,11 @@ instance : char_zero ℤ_[p] :=
 suffices (z1 : ℚ_[p]) = z2 ↔ z1 = z2, from iff.trans (by norm_cast) this,
 by norm_cast
 
-/--
-A sequence of integers that is Cauchy with respect to the `p`-adic norm
-converges to a `p`-adic integer.
--/
+/-- A sequence of integers that is Cauchy with respect to the `p`-adic norm converges to a `p`-adic
+integer. -/
 def of_int_seq (seq : ℕ → ℤ) (h : is_cau_seq (padic_norm p) (λ n, seq n)) : ℤ_[p] :=
 ⟨⟦⟨_, h⟩⟧,
- show ↑(padic_seq.norm _) ≤ (1 : ℝ), begin
+ show (padic_seq.norm _ : ℝ) ≤ (1 : ℝ), begin
```

Comment:
I don't see how writing down the explicit type for the cast is any better.

Reply:
Yeah, here since it's on the other side of the inequality it's not really an improvement. Adidtionally, the new spelling doesn't actually make it obvious that a cast is occuring, which probably makes it worse.

Reply:
This PR is still WIP; this one is going to be reverted when I have the time.